Homework10/Homework10/Repositories/ElectricalAppliancesRepository.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Homework10.Entities;
+using Homework10.Repositories.Abstractions;
+namespace Homework10.Repositories
+{
+	public class ElectricalAppliancesRepository : IElectricalAppliancesRepository
+	{
+		private ElectricalApplianceEntity[] _electricalAppliances = new ElectricalApplianceEntity[7];
+
+        public ElectricalApplianceEntity[] GenerateElectricalAppliance()
+        {
+            _electricalAppliances[0] = new RefrigeratorEntity()
+            {
+                PowerConsumption = 55,
+                Name = "Bosch",
+                IsPluggedIn = false,
```

Comment:
false is the default value for bool; therefore, you can just skip it and it will have 'false' value